app/models/station.rb

```diff
@@ -1,4 +1,3 @@
 class Station < ActiveRecord::Base
-  reverse_geocoded_by :latitude, :longitude, address: :location
-  after_validation :reverse_geocode
```

Comment:
It at least lets the Geocoder module know which attributes of the object to use with its included methods, but `after_validation` makes an api request, which I both don't need (because the station objects come with their own latlongs), and bumps up against the query limit many times in the process. Just seems unnecessary.